TF_Tooling_Web/src/TouchFree.ts

```diff
@@ -0,0 +1,24 @@
+import { ConnectionManager } from "./Connection/ConnectionManager";
+import { SVGCursor, SVGCursorParams } from "./Cursors/SvgCursor";
+import { TouchlessCursor } from "./Cursors/TouchlessCursor";
+import { WebInputController } from "./InputControllers/WebInputController";
+
+var InputController: WebInputController;
+export var CurrentCursor : TouchlessCursor;
+
+export function Init(_cursor: TouchlessCursor): void;
+export function Init(_svgParams?: SVGCursorParams): void;
```

Comment:
Overloads are unnecessary

Reply:
These are gone now, we have one function sig with an interface as discussed elsewhere